src/MetaCompilation.Analyzers/UnitTests/MetaCompilation.Analyzers.UnitTests.csproj

```
@@ -1,7 +1,7 @@
 ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

❗️ We don't want to _drop_ .NET Framework testing, but rather add .NET Core testing via multitargeting.

```suggestion
    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
```